2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_setup.py:_flush():69] setting env: {}
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Waste-Classification/wandb/run-20211001_154105-2r5bqbqo/logs/debug.log
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Waste-Classification/wandb/run-20211001_154105-2r5bqbqo/logs/debug-internal.log
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:init():375] calling init triggers
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-10-01 15:41:05,317 INFO    MainThread:116866 [wandb_init.py:init():397] re-initializing run, found existing run on stack: 3j9sqii8
2021-10-01 15:41:05,318 INFO    MainThread:116866 [wandb_run.py:finish():1271] finishing run ranuga-d/Waste-Classification/3j9sqii8
2021-10-01 15:41:05,318 INFO    MainThread:116866 [jupyter.py:save_history():353] not saving jupyter history
2021-10-01 15:41:05,319 INFO    MainThread:116866 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-01 15:41:05,319 INFO    MainThread:116866 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-10-01 15:41:05,319 INFO    MainThread:116866 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-01 15:41:05,319 INFO    MainThread:116866 [wandb_run.py:_restore():1543] restore
2021-10-01 15:41:06,244 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-10-01 15:41:07,252 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-10-01 15:41:08,518 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24034
}

2021-10-01 15:41:08,620 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:41:08,722 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:41:08,824 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:41:08,925 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:41:09,027 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:41:09,128 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:41:09,229 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 24115
}

2021-10-01 15:41:09,330 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8374
  total_bytes: 24115
}

2021-10-01 15:41:09,432 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:09,535 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:09,636 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:09,737 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:09,838 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:09,940 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:10,041 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:10,143 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:11,199 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:12,106 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:12,208 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24115
  total_bytes: 24115
}

2021-10-01 15:41:13,304 INFO    MainThread:116866 [wandb_run.py:_show_files():1915] logging synced files
2021-10-01 15:41:13,309 INFO    MainThread:116866 [wandb_init.py:init():424] starting backend
2021-10-01 15:41:13,309 INFO    MainThread:116866 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-01 15:41:13,311 INFO    MainThread:116866 [backend.py:ensure_launched():135] starting backend process...
2021-10-01 15:41:13,357 INFO    MainThread:116866 [backend.py:ensure_launched():140] started backend process with pid: 130442
2021-10-01 15:41:13,358 INFO    MainThread:116866 [wandb_init.py:init():429] backend started and connected
2021-10-01 15:41:13,370 INFO    MainThread:116866 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-01 15:41:13,371 INFO    MainThread:116866 [wandb_init.py:init():477] updated telemetry
2021-10-01 15:41:13,371 INFO    MainThread:116866 [wandb_init.py:init():500] communicating current version
2021-10-01 15:41:14,454 INFO    MainThread:116866 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-01 15:41:14,455 INFO    MainThread:116866 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-01 15:41:15,047 INFO    MainThread:116866 [wandb_init.py:init():540] starting run threads in backend
2021-10-01 15:41:20,053 INFO    MainThread:116866 [wandb_run.py:_console_start():1601] atexit reg
2021-10-01 15:41:20,053 INFO    MainThread:116866 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-01 15:41:20,053 INFO    MainThread:116866 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-01 15:41:20,054 INFO    MainThread:116866 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-01 15:41:20,054 INFO    MainThread:116866 [wandb_init.py:init():565] run started, returning control to user process
2021-10-01 15:41:32,606 INFO    MainThread:116866 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-01 15:41:32,607 INFO    MainThread:116866 [wandb_run.py:_restore():1543] restore
2021-10-01 15:41:33,576 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-10-01 15:41:34,154 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-10-01 15:41:35,119 INFO    MainThread:116866 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 8835
}

